neonvm/main.go

```diff
@@ -50,6 +56,11 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func logPanic(err error, msg string, keysAndValues ...any) {
+	setupLog.Error(err, msg, keysAndValues...)
+	panic(msg)
+}
+
```

Comment:
Can this be split into a separate PR?

Reply:
I see your point and it's good to separate different changes in different PRs, but at the same time having a different PR for every small change is too much. I'll revise the PR message to elaborate on logging here.